src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -250,7 +250,9 @@ public override async Task<bool> Execute()
                 {
                     if (await this.LoadGitHubClient(true))
                     {
-                        return commandEvent.IsSuccessful = await this.GitHubSubmitManifests(manifests);
+                        return commandEvent.IsSuccessful = await this.GitHubSubmitManifests(
+                            manifests,
+                            $"New package: {manifests.InstallerManifest.PackageIdentifier} version {manifests.InstallerManifest.PackageVersion}");

```

Comment:
I don't think this is always accurate, since the `wingetcreate new` command can also be used to add new versions of an existing package

Reply:
Perhaps you're confusing YamlCreate's behavior with wingetcreate? `wingetcreate new` will fail if it detects a PackageId that already exists in the repo
![image](https://github.com/microsoft/winget-create/assets/88161975/f9a14494-6762-45d8-b938-9ce8911e6332)

Reply:
> Perhaps you're confusing YamlCreate's behavior with wingetcreate? `wingetcreate new` will fail if it detects a PackageId that already exists in the repo ![image](https://user-images.githubusercontent.com/88161975/246883520-f9a14494-6762-45d8-b938-9ce8911e6332.png)

Really? That's interesting, I could have sworn I saw a few PR's before where `wingetcreate new` was used to create an update

Reply:
Am I missing something here? 😅 How did you know if the package was made with `wingetcreate new` or `wingetcreate update` from the PR title/contents/schema header? Cause wingetcreate doesn't really give any hints about that by default.